src/geo/ui/infowindow-view.js

```diff
@@ -30,6 +30,7 @@ var Infowindow = View.extend({
   events: {
     // Close bindings
     'click .js-close': '_closeInfowindow',
+    'click .cartodb-popup-close-button.close': '_closeInfowindow',
```

Comment:
Why not just `.close`? Also I'd add a comment explaining why need support for this selector, so that our future selves know why...